3 - PB/Documentazione esterna/Manuale Utente/Manuale Utente.typ

```diff
@@ -276,6 +276,37 @@ I parametri modificabili sono gli stessi che vengono richiesti durante la creazi
 
 Successivamente all'inserimento dei dati rappresentativi della zona modificata, è possibile premere sul pulsante "Salva le modifiche alla Zona" per aggiornare, come richiesto, l'elemento 3D corrispondente nell'ambiente di lavoro.
 
+== Ispezione Bin, Prodotti e Zone
+
+#figure(
+    grid(
+        columns: 3,
+        rows:    (auto, auto),
+        [ #image("./imgs/ispezione_1.png", width: 80%)],
+        [ #image("./imgs/ispezione_2.png", width: 100%)],
+        [ #image("./imgs/ispezione_3.png", width: 75%)],
+    ),caption: [Menù di ispezione zone (sinistra e centro) e menù di ispezione bin (destra)],
+) <ispezione>
+
+È possibile in ogni momento ispezionare una zona, facendo doppio click sul quadratino rosso nell'angolo in basso usato anche per lo spostamento, si aprirà quindi il menù in figura @ispezione (sinistra e centro), dove vengono riportate in ordine:
+- id della zona;
+- orientamento e dimensioni;
+- lista di bin che riporta per ogni voce:
+  - id del bin;
+  - stato (libero o se occupato riporta il nome del prodotto contenuto);
+  - un pulsante che (se occupato) mostra il contenuto del bin con il menù in @ispezione (destra).
```

Comment:
Riformulerei in: "pulsante per la visualizzazione dei dettagli del bin (menu @ispezione a destra), presente solo se il bin è occupato."